team13-fydp/fydp
# Update variables to make more dynamic

Updated variables to make more calculations  rather then to hard code. 

The full list of variable names and definitions is here: https://docs.google.com/spreadsheets/d/1I1KHVIqB4YNfxGNgKyVM6GDDSSSbLk4ttfBxU9c3HVU/edit#gid=0

The column "Changes in java- alyssa" describes any changes made. The cells highlighted in green mean that we need to account for these changes in our excel input/keep them in mind.  The cells in orange are the variables that were modified.

To test: 
1. comment out the model starting at the "try" line, and ending after the "catch" statement.
2. add the following block of code before the "try" statement and run.
System.out.println("frenchTeachlb: "+frenchTeachlb);
		System.out.println("frenchTeachub: "+frenchTeachub);
		System.out.println("frenchCohortlb: "+frenchCohortlb);
		System.out.println("frenchNum: " + frenchNum);
		System.out.println("cohortRange: " + cohortRange);
		System.out.println("subjectRange: " + subjectRange);
		System.out.println("primary: " + primary);
		System.out.println("timeChoice: " + timeChoice);
		for(int i=0; i<lengtht.length; i++) {
			System.out.print(lengtht[i]);
		}
3. This statement prints all of the variables changed. compare each value printed to their original values to ensure the changes are consistent.
